README.md

```diff
@@ -2,6 +2,8 @@
 
 [![tests](https://github.com/C5T/Current/actions/workflows/tests.yml/badge.svg?branch=stable)](https://github.com/C5T/Current/actions/workflows/tests.yml?query=branch%3Astable) [![cmake](https://github.com/C5T/Current/actions/workflows/cmake.yml/badge.svg?branch=stable)](https://github.com/C5T/Current/actions/workflows/cmake.yml?query=branch%3Astable)
 
+As of April 2024, Current has seen **10000 pull requests**. And counting!
```

Comment:
10'000 or 1'000? 🙂

Reply:
Fixed! It was `10'000`, and then my cold and rusty fingers failed the edit :-/